spec/models/comment_spec.rb

```diff
@@ -44,7 +44,7 @@
     end
 
     it "converts body_markdown to proper processed_html" do
-      expect(comment.processed_html.include?('<h1 id="hello-hy-hey-hey">')).to eq(true)
+      expect(comment.processed_html.include?('<h1 id="hello">')).to eq(true)
```

Comment:
The original test tested against `<h1>`, which would fail if the markdown text was
```markdown
# hello

hy hey hey
```
This change just includes the anchor id in an otherwise unrelated test.